Mathlib/Order/Chain.lean

```diff
@@ -93,10 +93,18 @@ theorem IsChain.image (r : α → α → Prop) (s : β → β → Prop) (f : α
   fun _ ⟨_, ha₁, ha₂⟩ _ ⟨_, hb₁, hb₂⟩ =>
   ha₂ ▸ hb₂ ▸ fun hxy => (hrc ha₁ hb₁ <| ne_of_apply_ne f hxy).imp (h _ _) (h _ _)
 
+lemma isChain_union {s t : Set α} :
+    IsChain r (s ∪ t) ↔ IsChain r s ∧ IsChain r t ∧ ∀ a ∈ s, ∀ b ∈ t, a ≠ b → r a b ∨ r b a := by
```

Comment:
Is the reverse implication worth `alias`ing for dot notation?

Reply:
I had considered this, but I don't think it's worth it because the third condition is sufficiently unwieldy